template_feed/Microsoft.DotNet.Test.ProjectTemplates.1.x/content/MSTest-VisualBasic/Company.TestProject1.vbproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' == ''">netcoreapp2.0</TargetFramework>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
+
+    <IsPackable Condition="'$(EnablePack)' == 'true'">true</IsPackable>

```

Comment:
Interesting, the VS templates didn't have it - https://github.com/dotnet/sdk/blob/master/src/Templates/ProjectTemplates/CSharp/.NETCore/CSharpUnitTest/ProjectTemplate.csproj. Any idea why it was added here? Do we basically instantiate it with EnablePack=false so that one can't pack a unit test project?